templates/annotate.sh

```diff
@@ -124,6 +127,72 @@ vep () {
   !{singularity_vep} vep "${args[@]}"
 }
 
+capice () {
+  local vcfPath="!{vcfPath}"
+  local vep_args=()
+  vep_args+=("--input_file" "${vcfPath}")
+  vep_args+=("--format" "vcf")
+  vep_args+=("--output_file" "${vcfPath%%.*}_prepared.vcf.gz")
+  vep_args+=("--vcf")
+  vep_args+=("--compress_output" "gzip")
+  vep_args+=("--regulatory")
+  vep_args+=("--sift" "s")
+  vep_args+=("--polyphen" "s")
+  vep_args+=("--domains")
+  vep_args+=("--numbers")
+  vep_args+=("--canonical")
+  vep_args+=("--symbol")
+  vep_args+=("--shift_3prime" "1")
+  vep_args+=("--allele_number")
+  vep_args+=("--no_stats")
+  vep_args+=("--offline")
+  vep_args+=("--cache")
+  vep_args+=("--dir_cache" "!{params.annotate_vep_cache_dir}")
+  vep_args+=("--species" "homo_sapiens")
+  vep_args+=("--assembly" "!{params.assembly}")
+  vep_args+=("--refseq")
+  vep_args+=("--use_given_ref")
+  vep_args+=("--exclude_predicted")
+  vep_args+=("--flag_pick_allele")
+  vep_args+=("--fork" "!{task.cpus}")
+  vep_args+=("--af_gnomad")
```

Comment:
probably not required (even though CAPICE documentation says so)

Reply:
You're right. Forgot to remove that one